# ZEO configuration file
%define INSTANCE_HOME /home/zope/instance
%define VIRTUAL_ENV /home/zope/venv

<zeo>
  # address 0.0.0.0:9999
  address  $INSTANCE_HOME/var/zeosocket
  read-only false
  invalidation-queue-size 100
  pid-filename $INSTANCE_HOME/var/ZEO.pid
  # monitor-address PORT
  # transaction-timeout SECONDS
</zeo>

<filestorage 1>
  path $INSTANCE_HOME/var/Data.fs
</filestorage>

<eventlog>
  level debug
  <logfile>
    path $INSTANCE_HOME/log/zeo.log
  </logfile>
</eventlog>

<runner>
  program $INSTANCE_HOME/bin/runzeo
  socket-name $INSTANCE_HOME/var/zeo.zdsock
  daemon true
  forever false
  backoff-limit 10
  exit-codes 0, 2
  directory $INSTANCE_HOME
  default-to-interactive true
  # user zope
  python $VIRTUAL_ENV/bin/python
  zdrun $VIRTUAL_ENV/lib/python2.7/site-packages/zdaemon/zdrun.py

  # This logfile should match the one in the zeo.conf file.
  # It is used by zdctl's logtail command, zdrun/zdctl doesn't write it.
  logfile $INSTANCE_HOME/log/zeo.log
</runner>
